nntrainer/tensor/blas_neon.cpp

```diff
@@ -276,6 +276,7 @@ void sgemv_transpose_neon(const float *A, const float *X, float *Y,
 
 void sgemv_neon_fp16(const __fp16 *A, const __fp16 *X, __fp16 *Y, uint32_t rows,
                      uint32_t cols, float alpha, float beta) {
+  const int batch = 0;
```

Comment:
Should this be 0 ?

Reply:
According to my current test, noTrans sgemv does not really bring significant round-off error with higher batches.
I think it is because there is more shrinking dimension per iter in SGEMV transpose than SGEMV noTrans.
However, it might cause potential error in the future (e.g. it might cause significant round-off error in unbalanced Tensors, sparse Tensors, or Tensors with big numbers) and we might some classification. Thus I temporally put zero in the code here for now

Reply:
Okay got it.